source/configure/push-notification-server-configuration-settings.rst

```diff
@@ -0,0 +1,99 @@
++-----------------------------------------------------------------+--------------------------------------------------------------------------------+
+
+Push notification server
+-------------------------
+
+|all-plans| |self-hosted|
+
+*Available in legacy Enterprise Edition E10/E20*
+
++-----------------------------------------------------------------+--------------------------------------------------------------------------------+
+| Location of Mattermost Push Notification Service (MPNS),        | - System Config path: **Environment > Push Notification Server**               |
+| which re-sends push notifications from Mattermost to services   | - ``config.json setting``: ``".EmailSettings.PushNotificationServer",``        |
+| like Apple Push Notification Service (APNS) and Google Cloud    | - Environment variable: ``MM_EMAILSETTINGS_PUSHNOTIFICATIONSERVER``            |
+| Messaging (GCM).                                                |                                                                                |
+|                                                                 |                                                                                |
+| - Customers running an Enterprise or Professional Edition       |                                                                                |
+|   workspace should enter ``https://push.mattermost.com`` for    |                                                                                |
+|   the push notification server hosted in the United States.     |                                                                                |
+| - If you prefer to use a push notification server hosted in     |                                                                                |
+|   Germany, enter ``https://hpns-de.mattermost.com/``.           |                                                                                |
+| - Team Edition customers should enter                           |                                                                                |
+|   ``https://push-test.mattermost.com``.                         |                                                                                |
++-----------------------------------------------------------------+--------------------------------------------------------------------------------+
+| **Note**:                                                                                                                                        |
+|                                                                                                                                                  |
+| - The TPNS is provided for testing push notifications prior to compiling your own service. Ensure you’re familiar with its `limitations          |
+|   <https://docs.mattermost.com/deploy/mobile-hpns.html#test-push-notifications-service-tpns>`__. Review the                                      |
```

Comment:
This link doesn't look like it's rendering.

Reply:
Interesting. I've confirmed it's working locally and in the preview. I'll also double-check once these changes are merged into production.